Userland/Utilities/headless-browser.cpp

```diff
@@ -155,7 +155,7 @@ class HeadlessBrowserPageClient final : public Web::PageClient {
     {
     }
 
-    virtual void page_did_start_loading(AK::URL const&) override
+    virtual void page_did_start_loading(AK::URL const&, bool) override
```

Comment:
This change should be included with the relevant commit (the first one I assume)